Add Acerola-Compute as a submodule

yDewolf opened the pull request:
# Fix: #1 Could not find type "ACompute" in the current scope. 
### This pull request also includes a input change I made to allow holding left click to change multiple cells

yDewolf committed:
A bit better ui inputs

yDewolf committed:
Add Acerola Compute as a git submodule

yDewolf committed:
trying to fix the submodule

yDewolf committed:
It's kinda fixed now

KwentiN-ui commented:
Thank you for this PR! I think adding the `--recurse-submodules` flag for git clone somewhere in the readme would avoid future confusion regarding a missing submodule after cloning. Also holding left click now plays the sound queue on repeat which is a bit irritating. It would be nice if it only played when a pixel is set or unset. I didn't find an immidiate fix for that in your changes but I could look into it if you don't feel like it.

yDewolf committed:
Fix sound queue on repeat

mentioned here: https://github.com/GarrettGunnell/God-Machine/pull/3#issuecomment-3245044000

yDewolf committed:
Update readme

yDewolf commented:
> Thank you for this PR! I think adding the `--recurse-submodules` flag for git clone somewhere in the readme would avoid future confusion regarding a missing submodule after cloning. Also holding left click now plays the sound queue on repeat which is a bit irritating. It would be nice if it only played when a pixel is set or unset. I didn't find an immidiate fix for that in your changes but I could look into it if you don't feel like it.

Thanks for the feedback, just fixed the sound queue and updated the readme

yDewolf commented:
Closed this PR since Acerola fixed the ACompute dependency
